### DISCUSSION TITLE
Prowler Cpu and Memory Usage

### DISCUSSION THREAD

**Discussion by Simon-Khoury** [prowler-cloud/prowler#2977] [Q&A] [answered]
for aws ecs fargate , how much cpu and memory needed when running security checks ?

**Comment by jfagoagas** [comment, marked as answer]
Hi @Simon-Khoury, this is a great question but this depends on the number of resources you have in your cloud accounts because Prowler does an intensive memory usage. I think that 1 vCPU and 2GiB would be enough for most uses cases.

Thanks for using Prowler 🚀
